test/unit/model/config-test.js

```diff
@@ -0,0 +1,28 @@
+import Model from 'controller/model';
+
+describe('Model', function() {
+    const config = {
+        file: '//playertest.longtailvideo.com/adaptive/bbbfull/bbbfull.m3u8',
+        image: 'http://d3el35u4qe4frz.cloudfront.net/bkaovAYt-480.jpg',
+        preload: 'auto'
+    };
+
+    let model;
+
+    before(function () {
+        model = new Model();
+        model.setup(config);
+    });
+
+    describe('return value', function() {
```

Comment:
@DanFerrer I would say that `describe` should be used for grouping related tests, usually the function being tested. Here, this could be `describe('Setup')`.